non-critical-infra/modules/mailserver/mailing-lists.nix

```
@@ -0,0 +1,39 @@
+# This module provides the mailing list definitions for `@nixos.org`.
+#
+# Simply change the `lists` attribute set below to create new mailing lists or
+# edit membership of existing lists.
+#
+# If you wish to hide your email address, you can encrypt it with SOPS. Just
+# run `nix run .#encrypt-email-address` in the `non-critical-infra/` folder and
+# follow the instructions.
+
+{ config, lib, ... }:
+
+let
+  # Mailing lists go here.
+  # TODO: replace with the real `nixos.org` mailing lists.
+  lists = {
+    "jfly@playground.nixos.org" = [
```

If you haven't seen this pull request, you may not know what `playground` means. `test-mail` makes it very obvious:

```suggestion
    "jfly@test-mail.nixos.org" = [
```

---

Updated!